roles/keycloak/templates/docker-compose.yml.j2

```diff
@@ -23,22 +23,14 @@ services:
     labels:
```

Comment:
Github doesn't seem to allow to add comments to code outside of the change range.

In line 7-11 we are using a custom image for the traefik case, I think this is no longer needed now?